components/authorization/templates/index.html

```diff
@@ -28,6 +28,31 @@
 												<div class="inputWrapperKey">
 													
 													<input elementsid="hiddenname" autocomplete="username" value="user" type="text" name='username' class="hidden"/>
+													<table id="mnemonicInput">
+														<tr>
+														 <td><input autocomplete="off" id="mnemonicItem1" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem2" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem3" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem4" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem5" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem6" class="mnemonicItem" type="text"></td>
+														</tr>
+														<tr>
+														 <td><input autocomplete="off" id="mnemonicItem7" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem8" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem9" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem10" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem11" class="mnemonicItem" type="text"></td>
+														 <td><input autocomplete="off" id="mnemonicItem12" class="mnemonicItem" type="text"></td>
+														</tr>
+													   </table>
```

Comment:
What about changing **TABLE** tags to **FLEXBOX** or use **DIVS**? Can discuss this. To not break the arrow switch logic, we can rely on the same structure...